package.json

```diff
@@ -21,9 +21,11 @@
     "broccoli-asset-rev": "^2.4.5",
     "broccoli-static-site-json": "^1.3.0",
     "ember-ajax": "^3.0.0",
+    "ember-browserify": "^1.2.1",
```

Comment:
Probably don’t need this any longer

Reply:
Ack! Yeah, sorry I missed that. I only ran npm uninstall.

Reply:
Ack! Yeah, sorry I missed that. I just ran npm uninstall.